src/sagas/MediaSaga.js

```diff
@@ -10,8 +11,8 @@ export function* addMediaSaga({ type, data }) {
       data.url,
       data.type,
       data.campaignId
-    )).media;
-
+    ));
+    newMedia.media.views = []; // Add views empty array so media can be checke
```

Comment:
Let's do this in the utils file so that the media object is properly serialized no matter where the function was called from.